tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -126,7 +126,7 @@ class DelegateProviders {
                      "XNNPACK delegate isn't supported on the platform!";
       } else {
         params_.Set<bool>("use_xnnpack", true);
-        params_.Set<bool>("num_threads", s.number_of_threads);
+        params_.Set<int32_t>("num_threads", s.number_of_threads);
```

Comment:
So it interfered xnnpack delegate option?

Reply:
Yes. Cannot be executed with the xnnpack delegate option.
```
Type mismatch while accessing parameter.
Abort
```


When specifying the xnnpack delegate option, the specification in `Interpreter::SetNumThreads` does not work. Must be specified in `DelegateProviders`.

Reply:
Got it. Thanks for the confirmation.